src/main/java/io/citrine/jcc/search/pif/query/PifSystemReturningQuery.java

```diff
@@ -120,6 +120,26 @@ public Boolean getAddLatex() {
     }
 
     /**
+     * Set whether to return the paths to extracted values.
+     *
+     * @param returnExtractedPath True to return extracted value paths.
+     * @return This object.
+     */
+    public PifSystemReturningQuery setReturnExtractedPath(final Boolean returnExtractedPath) {
```

Comment:
How expensive is it to compute the path?  Would we consider just always including it?

Reply:
It's not expensive. This is consistent with the optionality that include/exclude other fields as well. For example, it is computationally free (payload size aside) to return the PIF system, but we still allow users to exclude it. We can have this option default to true on the server.